### PR TITLE
Make the bot usable on startup without configuration

### LINKED ISSUE #352: Exception at startup if gitter token not provided
Opened by @jayvdb at [Gitter](https://gitter.im/coala/corobo?at=5a156c85982ea2653faf51dd)

### PR DESCRIPTION
**Pull request by jayvdb**
Fixes #352, #353 & #354

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 if not os.environ.get('COBOT_PREFIX'):
     BOT_ALT_PREFIXES = ('cobot ', )
 
-BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('meetmangukiya', )
+BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('@corobo', )
```
Comment (meetmangukiya) [inline config.py]: current `corobo` is actually @co-robo and the bot account shouldn't be its own admin IMO.

**Comment by jayvdb** [comment]
You're kidding me?  You are using this as the default for the production server?

Is there a way I can detect the production server (hostname?) in order to not break it?

File: config.py (hunk 1 above)
Comment (jayvdb) [reply config.py]: This value has nothing to do with the account `@co-robo` used by a production instance.

This is what other people 's test instance has as a value for this field.

I am a bit interested in whether the `@` there causes any problems.

**Comment by yukiisbored** [comment]
@jayvdb No, there's no way to check if it's broken or not other than testing corobo. Due to the amount of RAM we got, we can't startup a whole grafana setup

File: config.py (hunk 1 above)
Comment (jayvdb) [reply config.py]: The `@` there does no harm, but according to the docs, `corobo@localhost` is going to be safer

File: config.py [hunk 2]
```diff
@@ -21,10 +21,10 @@
 if not os.environ.get('COBOT_PREFIX'):
     BOT_ALT_PREFIXES = ('cobot ', )
 
-BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('meetmangukiya', )
+BOT_ADMINS = os.environ.get('BOT_ADMINS', '').split() or ('*@localhost', )
```
Comment (meetmangukiya) [inline config.py]: According to errbot specs, for text backend, it should be `@localhost` or `#localhost`

**Comment by meetmangukiya** [comment]
ack c79e25dbe90300dc168c30ece7e8d103ef3361a0 b8c691fae374311bd391d17e15104fe6b2057d11

File: config.py (hunk 2 above)
Comment (jayvdb) [reply config.py]: No it isnt.  See `config-template.py`

Comment (jayvdb) [reply config.py]: See also https://github.com/errbotio/errbot/issues/1135

Comment (meetmangukiya) [reply config.py]: okk :+1:

**Comment by meetmangukiya** [comment]
ack da63e1024ccda0e7df466630240b150ad5f63f0f

**Comment by meetmangukiya** [comment]
@rultor merge

File: config.py (hunk 2 above)
Comment (jayvdb) [reply config.py]: Sorry, you were right for the `Text` backend.  We should fix that here quickly, and upstream, as it is silly that `Text` doesnt accept the normal syntax when it shouldnt care about the bot admin anyway as it isnt on a shared service.

**Pull request merged**